env.go

```diff
@@ -130,3 +132,44 @@ func runUnset(cmd *Command, args []string) {
 	must(err)
 	log.Printf("Unset env vars and restarted %s.", appname)
 }
+
+var cmdLoad = &Command{
+	Run:      runLoad,
+	Usage:    "load <file>",
```

Comment:
Can we change this to env-load just to make it a little more specific & matching the rest of the CLI?